new/LIT6217MGEulogius.xml

```diff
@@ -0,0 +1,105 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6217MGEulogius" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of St George: the impoverished ship owner
+                    Eulogius recovered his money from the Egyptian thief</title>
```

Comment:
Again recovers to keep tense consistent?

Reply:
already changed with the second PR. Thank you!!!